source/atlas-vector-search.txt

```diff
@@ -0,0 +1,178 @@
+``$vectorSearch`` stage to perform a vector search on the
+``sample_mflix.embedded_movies`` collection in the :atlas:`Atlas sample datasets
+</sample-data>`.
+
+To learn how to create a free MongoDB Atlas cluster and load the sample datasets, see the
+:atlas:`Get Started with Atlas </getting-started>` guide.
+
+Vector Search Example
+~~~~~~~~~~~~~~~~~~~~~
+
+You can perform a vector search query by using the ``$vectorSearch`` stage
+in an :ref:`aggregation pipeline <node-aggregation>`. To perform a vector
+search on a collection, you must first have a collection with a field that contains
+vector embeddings and a vector search index that covers that field. 
+
+In the following example, the aggregation pipeline searches the ``plot``  field of each
+document in the collection for text semantically related to the term "time travel". The
+``queryVector`` field in the ``$vectorSearch`` pipeline is the vector representation of
+your query.
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: /includes/aggregation/atlas-vector-search-example.js
+      :language: javascript
+      :dedent:
+
+   .. output:: 
+      :language: json
+      :visible: false
+
+      {
+      plot: 'A reporter, learning of time travelers visiting 20th century disasters, tries to change the history they know by averting upcoming disasters.',
+      title: 'Thrill Seekers',
+      score: 0.9259490966796875
+      }
+      {
+      plot: 'At the age of 21, Tim discovers he can travel in time and change what happens and has happened in his own life. His decision to make his world a better place by getting a girlfriend turns out not to be as easy as you might think.',
+      title: 'About Time',
+      score: 0.9253997802734375
+      }
+      {
+      plot: 'An officer for a security agency that regulates time travel, must fend for his life against a shady politician who has a tie to his past.',
+      title: 'Timecop',
+      score: 0.922332763671875
+      }
+      {
+      plot: 'After using his mother's newly built time machine, Dolf gets stuck
+      involuntary in the year 1212. He ends up in a children's crusade where he confronts
+      his new friends with modern techniques...',
+      title: 'Crusade in Jeans',
+      score: 0.92205810546875
+      }
+      {
+      plot: 'Hoping to alter the events of the past, a 19th century inventor instead travels 800,000 years into the future, where he finds humankind divided into two warring races.',
+      title: 'The Time Machine',
+      score: 0.921875
+      }
+      {
+      plot: 'A time-travel experiment in which a robot probe is sent from the year 2073 to the year 1973 goes terribly wrong thrusting one of the project scientists, a man named Nicholas Sinclair into a...',
+      title: 'A.P.E.X.',
+      score: 0.9202728271484375
+      }
+      {
+      plot: 'Agent J travels in time to M.I.B.'s early days in 1969 to stop an alien from assassinating his friend Agent K and changing history.',
```

Comment:
```suggestion
      plot: "Agent J travels in time to M.I.B.'s early days in 1969 to stop an alien from assassinating his friend Agent K and changing history.",
```

This string is broken because there's a single quote inside of it when its being quoted by single quotes